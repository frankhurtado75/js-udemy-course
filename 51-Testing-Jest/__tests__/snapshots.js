// Para actualizar snapshot: npm run test -- -u
const cliente = {
    nombre: 'Franklin GaBriel',
    balance: 500,
    tipo: 'Premium'
};

describe('Testing al cliente', () => {
    test('Es Franklin 2', () => {
        // Convierte el objeto en un string en otro archivo
        expect(cliente).toMatchSnapshot();
    });
});