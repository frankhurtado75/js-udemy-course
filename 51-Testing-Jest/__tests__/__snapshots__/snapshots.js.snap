// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Franklin 2 1`] = `
Object {
  "balance": 500,
  "nombre": "Franklin GaBriel",
  "tipo": "Premium",
}
`;
